addons/website/static/src/snippets/s_chart/chart.edit.js

```diff
@@ -1,16 +1,17 @@
 import { Chart } from "./chart";
 import { registry } from "@web/core/registry";
 
-const ChartEdit = I => class extends I {
-    setup() {
-        super.setup();
-        this.noAnimation = true;
-    }
-};
+const ChartEdit = (I) =>
+    class extends I {
+        setup() {
+            super.setup();
+            this.noAnimation = true;
+            this.websiteEditService = this.services.website_edit;
+            this.websiteEditService.callShared("builderOverlay", "refreshOverlays");
+        }
```

Comment:
For this one, it should be in `start` not `setup`, because it is `start` that redraws :wink: (as you can see by testing, your fix didn't work well with the chart :+1: )